pr2eus/robot-interface.l

```diff
@@ -44,6 +44,58 @@
   (:time-to-finish ()
     (ros::ros-debug "[~A] time-to-finish ~A" ros::name-space time-to-finish)
     time-to-finish)
+  (:action-result-cb (msg)
+    "
+    display error messages from [depending on robots]/follow_joint_trajectory/result
+    "
+    ;; The following error messages will not appear for the first (send *ri* :angle-vector)
+    ;; To display it, use :wait-interpolation or (ros::spin-once groupname)
+    (let ((status-num (send (send msg :status) :status))
+          (status-word '(PENDING
+                         ACTIVE
+                         PREEMPTED
+                         SUCCEEDED
+                         ABORTED
+                         REJECTED
+                         PREEMPTING
+                         RECALLING
+                         RECALLED
+                         LOST))
+          (status-text (send (send msg :status) :text))
+          (error-code nil)
+          (error-word '(SUCCESSFUL
+                        INVALID_GOAL
+                        INVALID_JOINTS
+                        OLD_HEADER_TIMESTAMP
+                        PATH_TOLERANCE_VIOLATED
+                        GOAL_TOLERANCE_VIOLATED)))
+      ;; status
+      (cond ((not (or (= status-num 3) (= status-num 8)))
```

Comment:
You can use `actionlib_msgs::GoalStatus::*succeeded*` and `actionlib_msgs::GoalStatus::*recalled*` instead of 3 and 8.
I think this is easier to read for human.
```
17.irteusgl$ actionlib_msgs::GoalStatus::*succeeded*
3
18.irteusgl$ actionlib_msgs::GoalStatus::*recalled*
8
```

Reply:
I think this is easier to read, too.
I'll use them.